sdcm/tester.py

```diff
@@ -2992,7 +2992,7 @@ def tag_ami_with_result(self):
         test_result = 'PASSED' if self.get_test_status() == 'SUCCESS' else 'ERROR'
         job_base_name = os.environ.get('JOB_BASE_NAME', 'UnknownJob')
         ami_id = self.params.get('ami_id_db_scylla').split()[0]
-        region_name = self.params.get('aws_region').split()[0]
+        region_name = self.params.get('region').split()[0]
```

Comment:
Please revert this one

Reply:
Won't this fail if we revert? I think we won't have the a `aws_region` param after the changes.

Reply:
Nope, your change related to jenkins files only, you did not touch sct_config.py.
Therefore this file should not be touched in the same commit.

But, thanks tou your comment, I found that it never worked due to the wrong parameter name.
Having that, could you please apply following change in separate commit:

```suggestion
        region_name = self.params.region_names[0]
```

Reply:
Ah, the wonders of code review :) I applied the requested change in a separate commit. I also added another commit with changes to the `vars` files to avoid naming conflicts.